sample-lnd.conf

```diff
@@ -846,6 +846,22 @@ litecoin.node=ltcd
 ; value must be >= 1m.
 ; healthcheck.diskspace.interval=6h
 
+; The number of times we should attempt to check for certificate expiration before
+; gracefully shutting down. Set this value to 0 to disable this health check.
+; healthcheck.tls.attempts=2
+
+; The amount of time we allow a query for certificate expiration to take
+; before we fail the attempt. This value must be >= 1s.
+; healthcheck.tls.timeout=5s
+
+; The amount of time we should backoff between failed attempts to query
+; certificate expiration. This value must be >= 1s.
+; healthcheck.tls.backoff=1m
+
+; The amount of time we should wait between certificate expiration health checks.
+; This value must be >= 1m.
+; healthcheck.tls.interval=6h
```

Comment:
Feels like a smaller number would be more sensible but not sure what. `1m`?

Reply:
Always hard to pick these magic numbers, happy with 1/2 mins.